app/view_models/profile_index.rb

```diff
@@ -62,10 +62,10 @@ def manage_personal_key_partial
   def header_personalization
     return decrypted_pii.first_name if decrypted_pii.present?
 
-    current_user.email
+    decorated_user.email
```

Comment:
I'd rather see explicit `delegate` call too, and not name the internal object `model` but `user` so that it is more clear what it is.